build.gradle

```diff
@@ -3,9 +3,9 @@
 plugins {
   id 'application'
   id 'java'
-  id 'com.bmuschko.clover' version '3.0.1'
   id 'net.nemerosa.versioning' version '2.8.2'
   id 'com.github.johnrengelman.shadow' version '6.0.0'
+  id 'jacoco'
```

Comment:
Do we not want to pin a version?

Reply:
Error resolving plugin [id: 'jacoco', version: '0.8.7']
> Plugin 'jacoco' is a core Gradle plugin, which cannot be specified with a version number. Such plugins are versioned as part of Gradle. Please remove the version number from the declaration.

Reply:
Actually, if you want, I can add a commit to sort the plugins so all the core plugins are in one block.